srv/DeviceReport.srv

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Could this be switched to `std_srvs/Trigger`?

Reply:
yup! changed all unnecessary services to trigger